### ISSUE TITLE
Remove optional chaining from code that will be transpiled (ipfs-http-client)

### ISSUE THREAD

**Issue by schettn** [ipfs/js-ipfs#3795]
Hi! 👋 
      
Firstly, thanks for your work on this project! 

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `ipfs-http-client@51.0.1` for the project I'm working on.

I use ipfs-http-client inside of https://github.com/storybookjs/storybook.
Due to optional chaining storybooks built in webpack configuration fails to build.

I am aware that this problem is mainly related to storybook however I saw the following PR #3698.
This topic is also taken up there.
It would be very nice to enjoy the next release of ipfs-http-client` without optional chaining.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/ipfs-http-client/src/index.js b/node_modules/ipfs-http-client/src/index.js
index 521aaa7..a2e90b4 100644
--- a/node_modules/ipfs-http-client/src/index.js
+++ b/node_modules/ipfs-http-client/src/index.js
@@ -42,7 +42,7 @@ function create (options = {}) {
     loadBase: options.ipld && options.ipld.loadBase
   })
   const codecs = new Multicodecs({
-    codecs: [dagPb, dagCbor, raw, json, id].concat(options.ipld?.codecs || []),
+    codecs: [dagPb, dagCbor, raw, json, id].concat(options.ipld && options.ipld.codecs || []),
     loadCodec: options.ipld && options.ipld.loadCodec
   })
   const hashers = new Multihashes({
```

**Comment by achingbrain** [comment]
Thanks for opening this - a fix for this went in as part of #3785 - could you please try with the latest rc?

```console
$ npm i ipfs-http-client@next
```

**Comment by achingbrain** [comment]
This was shipped in `ipfs-http-client@52.0.0`, please re-open if you're still having problems.
